test/functional/change_stream_spec_tests.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+
+var setupDatabase = require('./shared').setupDatabase;
+const chai = require('chai');
+const fs = require('fs');
+const expect = chai.expect;
+const camelCase = require('lodash.camelcase');
+chai.use(require('chai-subset'));
+
+describe.only('Change Stream Spec', function() {
+  const specStr = fs.readFileSync(`${__dirname}/spec/change-stream/change-streams.json`, 'utf8');
+  const specData = JSON.parse(specStr);
+  const ALL_DBS = [specData.database_name, specData.database_name_2];
+
+  const delay = ms => new Promise(r => setTimeout(r, ms));
+
+  before(function() {
+    const MongoClient = this.configuration.require.MongoClient;
```

Comment:
I know we don't have a hard-set rule for this, but can we just directly include `MongoClient` above. I'm uncomfortable using the magic `configuration` more than absolutely necessary